public/scripts/app.js

```diff
@@ -2,6 +2,144 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+    //  COUNTER CLOCK - RESETS EVERYTIME PAGE IS REFRESHED
+    let startDateTime = new Date();
+    let startStamp = startDateTime.getTime();
 
+    function updateClock() {
+        let newDate = new Date();
+        let diff = Math.round(newDate-startStamp);
+        let days = Math.floor(diff/(1000*60*60*24));
+        let hours = Math.floor(diff/(1000*60*60));
+        let minutes = Math.floor(diff/(1000*60));
+        let seconds = Math.floor(diff/1000);
+
+        document.getElementById("time-elapsed").innerHTML = days +" day(s), "+ hours +" hour(s), "+ minutes +" minute(s), "+ seconds +" second(s) wasting away without pie";
+    };
+
+    setInterval(updateClock, 1000); // clock changes by 1 second
+
+    // GET ALL PIES ON PAGE LOAD
+    let template;
+    let $pieList = $('.pie-list');
+    let allPies = [];
+
+    // display on pie donations as HTML
+    $.ajax({
+        method: "GET",
+        url: "/api/pies",
+        success: function(pies) {
+            // SET 'allPies' to pie data from API
+            pies.forEach(function (pie) {
+              $(".pie-list").append(`
+                <div class="pie" id="pie-${pie._id}">
+                  <p>${pie.donor_name} sent Bear ${pie.pie_quantity} ${pie.pie_type} pie(s)</p><img src="https://bearheartsbat.files.wordpress.com/2016/06/bouncypiebear.gif?w=1142" height="auto" width="200"><br><br>
+                </div>
+              `);
+            })
+        },
+        error: function() {
+          alert("Pie Error! :(");
+        }
+
+    })
+    //serialize form data
+    var newPie = $(this).serialize();
+    $pieList.on('submit', function(e) {
```

Comment:
instead of `$pieList`, you wanted to listen for submission on your form. You could do this with `$('form').on('submit', ...`, or by giving your form a class and then something like `$('.newPieForm').on('submit',...`